keyboards/ergodox/keymaps/narze/keymap.c

```diff
@@ -0,0 +1,852 @@
+      return false;
+      break;
+    case PLOVER:
+      if (record->event.pressed) {
+        layer_off(_RAISE);
+        layer_off(_LOWER);
+        layer_off(_ADJUST);
+        layer_on(_PLOVER);
+        if (!eeconfig_is_enabled()) {
+            eeconfig_init();
+        }
+        keymap_config.raw = eeconfig_read_keymap();
+        keymap_config.nkro = 1;
+        eeconfig_update_keymap(keymap_config.raw);
+      }
+      return false;
+      break;
+    case EXT_PLV:
+      if (record->event.pressed) {
+        layer_off(_PLOVER);
+      }
+      return false;
+      break;
+    case SDTOGG:
+      if (record->event.pressed) {
+        // superduper_enabled = !superduper_enabled;
+
+        // if (superduper_enabled) {
+        //   set_superduper_key_combos();
+        // } else {
+        //   clear_superduper_key_combos();
+        // }
+      }
+      return false;
+      break;
+    case EPRM:
+      if (record->event.pressed) {
+        eeconfig_init();
+      }
+      return false;
+      break;
+    case VRSN:
+      if (record->event.pressed) {
+        SEND_STRING (QMK_KEYBOARD "/" QMK_KEYMAP " @ " QMK_VERSION);
+      }
+      return false;
+      break;
+    case RGB_SLD:
+      if (record->event.pressed) {
+        #ifdef RGBLIGHT_ENABLE
+          rgblight_mode(1);
+        #endif
+      }
+      return false;
+      break;
+  }
+  return true;
+}
+
+void matrix_init_user(void) {
+
+}
+
+void matrix_setup(void) {
+  // set_superduper_key_combos();
+}
+
+// void set_superduper_key_combos(void) {
+//   uint8_t layer = eeprom_read_byte(EECONFIG_SUPERDUPER_INDEX);
+
+//   switch (layer) {
+//     case _QWERTY:
+//     case _COLEMAK:
+//     case _QWOC:
+//       key_combos[CB_SUPERDUPER].keys = superduper_combos[layer];
+//       break;
+//   }
+// }
+
+// void clear_superduper_key_combos(void) {
+//   key_combos[CB_SUPERDUPER].keys = empty_combo;
+// }
+
+void matrix_scan_user(void) {
+  // uint8_t layer = biton32(layer_state);
+
+  //   ergodox_board_led_off();
+  //   ergodox_right_led_1_off();
+  //   ergodox_right_led_2_off();
+  //   ergodox_right_led_3_off();
+  //   switch (layer) {
+  //     // TODO: Make this relevant to the ErgoDox EZ.
+  //       case 1:
+  //           ergodox_right_led_1_on();
+  //           break;
+  //       case 2:
+  //           ergodox_right_led_2_on();
+  //           break;
+  //       default:
+  //           // none
+  //           break;
+  //   }
+}
+
+// Combos
+
+// void process_combo_event(uint8_t combo_index, bool pressed) {
+//   if (pressed) {
+//     switch(combo_index) {
+//       case CB_SUPERDUPER:
+//         layer_on(_SUPERDUPER);
+//         ergodox_board_led_on();
+//         break;
+//     }
+//   } else {
+//     layer_off(_SUPERDUPER);
+//     ergodox_board_led_off();
+//     unregister_mods(MOD_BIT(KC_LGUI) | MOD_BIT(KC_LCTL) | MOD_BIT(KC_LALT)); // Sometimes mods are held, unregister them
+//   }
+// }
+
+// Macros
+
+const uint16_t PROGMEM fn_actions[] = {
```

Comment:
Could you remove the `fn_actions` block?  All of it's features have been added as keycodes, and been deprecated in favor of `process_record_user`.